packages/create-resolve-app/src/create_resolve_app.js

```diff
@@ -0,0 +1,189 @@
+import spawn from 'cross-spawn';
```

Comment:
Does this file appears to be major copy-paste from here: https://github.com/facebookincubator/create-react-app/blob/master/packages/create-react-app/createReactApp.js, with refactor to ES6 language ? 

Or source, such code is excepted solution, but what about licences? 

```
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 */
```

If all ok, not problem